### DISCUSSION TITLE
Suricata - unit suricata.service not found error

### DISCUSSION THREAD

**Discussion by deutschmannz** [Security-Onion-Solutions/securityonion#11073] [Unsupported Versions] [answered]
Hello everyone,
I am looking for opinions, advice or help in diagnosing an error we are receiving for Suricata, specifically inside Security Onion. In our server, we are receiving the error “unit suricata.service not found” when executing “systemctl status suricata.service”.

We are looking to start/restart suricata from Linux, and when executing so-suricata-start, we receive this error:

We also do not see Suricata listed as a service under a docker container, so we are wondering if suricata has been enabled as a service on a container or its own standalone service.

Overall, we are curious as to why we cannot tune/suppress alerts from inside our terminal, but can see alerts generate on security onion. Any advice, tips/tricks, or ways to troubleshoot this issue is greatly appreciated. Thank you!

**Comment by dougburks** [comment, marked as answer]
Please provide more information as shown in https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Please include the following details where you can:
Security Onion version as seen in the lower left corner of SOC and in /etc/soversion. For example: 2.4.10
Is this a cloud deployment or on-prem? If on-prem, do you have Internet access or this an airgap installation?
Did you install from our Security Onion ISO image or did you perform a network installation?
If network installation, what distro and version did you install on?
How many nodes do you have?
What are the hardware specs of each of those nodes?
How are each of those nodes configured? (ex. manager with 2 search nodes and 3 forward nodes)
Are you experiencing issues monitoring network traffic? If so, are you sniffing from a tap or span port and what is the traffic volume?
Does so-status show all services running?
Do you get any failures when you run sudo salt-call state.highstate?
Does the SOC Grid page show any failures?
Explain your issue. For example: Installation fails when I select this series of options...
Provide applicable logs. If you are having problems right after setup, provide /root/sosetup.log. If you are having problems during soup, provide /root/soup.log. If you are having problems with a specific component, provide that component's logs from /opt/so/log/.
